setup.py

```diff
@@ -212,7 +213,7 @@ def build_dependency(lib_name):
 
 setuptools.setup(
     name="aws_crt",
-    version="0.0.1",
+    version="0.2.2",
```

Comment:
Shouldn't this be 0.2.0?

Reply:
no, it should match the current tag, which will be cut as soon as I merge this.

Reply:
I should have looked, no worries.